libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -765,6 +749,27 @@ bool AP_Follow::handle_follow_target_message(const mavlink_message_t &msg)
         _target_heading_rate_degs = 0.0f;
     }
 
+    _target_pos_ned_m = target_pos_neu_m; // convert from cm to meters
```

Comment:
The comment is wrong, or at least not what the line does.

Reply:
Oh crap!